REDCapUITweaker.php

```diff
@@ -290,6 +385,48 @@ function redcap_every_page_top( $project_id = null )
 		}
```

Comment:
Minor remark on existing code (that GitHub doesn't seem to want to allow me to put a comment on)
I'm not a fan of literal numbers in code, as so easy to get wrong - especially if updated during maintenance.
It occurs to me a safer way of doing this kind of thing in the lines above would be:
instead of
strlen( APP_PATH_WEBROOT ), 22 ) == 'ProjectSetup/index.php'
do
strlen( APP_PATH_WEBROOT ), strlen('ProjectSetup/index.php' ) == 'ProjectSetup/index.php'
That way if the string we are checking is ever changed, the programmer doesn't have to count the letters themselves.
On the other hand, if this is running on every page, then the existing option may be better from a performance point of view.
so... up to you if you want to change or not.

Reply:
I see what you're saying, but I'm not sure replacing a literal number with a duplication of a literal string really makes a lot of difference.
If I change these, it's probably better to utilise constants, or change the condition to something like the following
`strpos( substr( PAGE_FULL, strlen( APP_PATH_WEBROOT ) ), 'ProjectSetup/index.php' ) === 0`
However, as you say, this could have performance implications.